hexrdgui/absorption_correction_options_dialog.py

```diff
@@ -10,7 +11,19 @@ def __init__(self, parent=None):
                                    parent)
 
         self.additional_materials = {}
-        self.filters = {n: {} for n in HexrdConfig().detector_names}
+        # Get filter defaults for each detector
+        self.filters = {}
+        for det in HexrdConfig().detector_names:
+            filter = HexrdConfig().detector_filter(det)
+            if filter is None:
+                HexrdConfig().update_detector_filter(det)
+                filter = HexrdConfig().detector_filter(det)
```

Comment:
So, if there isn't a filter, we also wouldn't want to add one unless the user clicks "OK", right? I think we shouldn't modify any settings unless the user clicks "OK".

In this case, if the user clicks "Cancel", would there now be a filter present for that detector?